packages/ra-data-simple-prisma/src/dataProvider.ts

```diff
@@ -52,8 +52,7 @@ export const dataProvider = (
       options.axiosInterceptors.response.forEach((value) =>
         apiService.interceptors.response.use(
           value.onFulfilled,
-          value.onRejected,
-          value.options
+          value.onRejected
```

Comment:
only accepts 2 arguments now. can options be emmited? @ogroppo

Reply:
well, it would be a breaking change for people using the package so major version bump... where are the options gone then?